src/toolkit-cleaner/index.ts

```diff
@@ -17,11 +17,13 @@ import { GetStackNamesFunction } from './get-stack-names-function';
  */
 export interface ToolkitCleanerProps {
   /**
-   * The schedule for the cleaner
+   * The schedule for the cleaner.
+   *
+   * Pass `null` to only run manually via the Step Functions interface
    *
    * @default - every day
    */
-  readonly schedule?: Schedule;
+  readonly schedule?: Schedule | null;
```

Comment:
Unfortunately we cannot do this kind of union types with `jsii` (see failing build).

Let's add a `enabled` prop and pass it to the `Rule` (which already defaults to `true` for this prop).

Reply:
Sounds good. Updates made!